rpcs3/rpcs3.cpp

```diff
@@ -96,7 +96,7 @@ cfg::map_entry<std::function<std::shared_ptr<GSRender>()>> g_cfg_gs_render(cfg::
 	{ "Null", PURE_EXPR(std::make_shared<NullGSRender>()) },
 	{ "OpenGL", PURE_EXPR(std::make_shared<GLGSRender>()) },
 #ifdef _MSC_VER
-	{ "DX12", PURE_EXPR(std::make_shared<D3D12GSRender>()) },
+	{ "DirectX 12", PURE_EXPR(std::make_shared<D3D12GSRender>()) },
```

Comment:
We do not use all DirectX in renderer, just Direct3D part. So that name incorrect too